tests/MilkCheckTests/EngineTests/ServiceGroupTest.py

```diff
@@ -636,13 +638,16 @@ def test_filter_from_service_group(self):
 
         grp = ServiceGroup('group')
         subsvc = Service('subsvc')
-        subsvc.add_action(Action('stop', command='true', target=nodes))
+        subsvc.add_action(Action('stop', command='/bin/true', target=nodes))
         grp.add_inter_dep(subsvc)
 
         svc = Service('svc')
-        svc.add_action(Action('stop', command='true', target=nodes))
+        svc.add_action(Action('stop', command='/bin/true', target=nodes))
         svc.add_dep(grp, sgth=FILTER)
-        svc.run('stop')
+
+        manager = ServiceManager()
```

Comment:
why is there a need to ServiceManager here? 
All tests in this class were using a group and a service and were fine. Why know these 2 tests need this different setup?

Reply:
I did not look at this code for a while :D but my guts is telling me there is something else wrong there. Mostly because we have other similar tests which are not touched, and also because the modified tests exist for a while.